src/Beerware.xml

```diff
@@ -0,0 +1,12 @@
+<SPDX name="Beerware License" identifier="Beerware" osi-approved="false">
+  <urls>
+    <url>https://fedoraproject.org/wiki/Licensing/Beerware</url>
+  </urls>
+  <license>
+    <body>
+      <p>"THE BEER-WARE LICENSE" (Revision 42): &lt;phk@FreeBSD.ORG&gt; wrote this file. As long as you retain
```

Comment:
The </p> is on line 9 -- the text can span multiple lines, which is how we keep it from scrolling way off to the side :)